lib/android/audio.nit

```diff
@@ -31,37 +31,49 @@ in "Java" `{
 	import android.media.SoundPool;
 	import java.io.IOException;
 	import android.media.AudioManager;
+	import android.media.AudioManager.OnAudioFocusChangeListener;
 	import android.content.Context;
 	import android.util.Log;
 `}
 
+in "Java inner" `{
+	static OnAudioFocusChangeListener afChangeListener = new OnAudioFocusChangeListener() {
+		public void onAudioFocusChange(int focusChange) {
+			if(focusChange == AudioManager.AUDIOFOCUS_LOSS_TRANSIENT) {
+			}else if (focusChange == AudioManager.AUDIOFOCUS_GAIN) {
+			}else if (focusChange == AudioManager.AUDIOFOCUS_LOSS) {
+			}
+		}
+	};
+`}
+
 # AudioManager of the application, used to manage the audio mode
 extern class NativeAudioManager in "Java" `{ android.media.AudioManager `}
 	super JavaObject
 
+	# Returns the current audio mode ( MODE_NORMAL = 0, MODE_RINGTONE = 1, MODE_IN_CALL = 2 or MODE_IN_COMMUNICATION = 3 )
```

Comment:
Please split the comment into two part:

1- Synopsis, one line less than 80 chars, containing the relevant part of the comment
2- The rest of the comment